react-game/src/redux/store.ts

```diff
@@ -0,0 +1,19 @@
+import {applyMiddleware, combineReducers, compose, createStore} from "redux";
+import thunkMiddleware from "redux-thunk";
+import gameReducer from "./game-reducer";
+
+let reducers = combineReducers({
```

Comment:
If U kinda know about combineReducers, U could make the structure of the store not such linear, that everything is inside `game` object. I mean more tree like structure, group by cards, volume, score ...